modules/data-wrangling/index.qmd

```diff
@@ -110,7 +78,10 @@ penguins |>
 
 ???
 
-!all(c("body_mass_g", "year") %in% all.vars(.code))
+c(
+    "Did you use the right function?" != search_ast(.code, .fn = select),
```

Comment:
This isn't working because you are comparing the hint with the result of `search_ast()`.

It should instead be: 

```suggestion
    "Did you use the right function?" = !search_ast(.code, .fn = select),
```